UKHomeOffice/nmsw-ui
# NMSW-155 Your details page for registration & an InputPhone component

# Ticket

NMSW-155

----

## AC

Given I provided my email address CORRECTLY
AND they match
WHEN I submit them
THEN I am redirected to provide my other details:

GIVEN I have provided ALL of my details
WHEN I Submit my details
THEN I am asked to create my password

----

## To test

- run locally
- click on create account link
- enter email addresses and continue
- > you should see the Your details page
- click submit
- > all fields should error
- enter something in fullname
- enter something in company name
- enter something in JUST THE SMALL FIELD for phone number, leave the large field blank
- enter something in country
- select if a shipping agent
- click submit
- > phone number field should error
- delete value from short field and enter something in long field for phone number
- click submit
- > phone number field should error
- enter something in both fields for phone number
- refresh page
- > values should persist 
- open your console
- click continue
- > form data should be viewable in console
- > password page should load

----

## Notes
